server/services/licenceService.ts

```diff
@@ -274,37 +274,15 @@ export default class LicenceService {
     return this.licenceApiClient.matchLicences(statuses, filteredPrisons, [], [], 'conditionalReleaseDate', null, user)
   }
 
-  async getLicencesForCaseAdmin(user: User): Promise<LicenceSummary[]> {
+  async getLicencesForOmu(user: User): Promise<LicenceSummary[]> {
     const statuses = [
       LicenceStatus.ACTIVE.valueOf(),
       LicenceStatus.APPROVED.valueOf(),
       LicenceStatus.REJECTED.valueOf(),
       LicenceStatus.SUBMITTED.valueOf(),
     ]
-    if (user.authSource === 'nomis') {
-      const filteredPrisons = filterCentralCaseload(user.prisonCaseload)
-      return this.licenceApiClient.matchLicences(
-        statuses,
-        filteredPrisons,
-        [],
-        [],
-        'conditionalReleaseDate',
-        null,
-        user
-      )
-    }
-    if (user.authSource === 'delius') {
```

Comment:
Took this out as it doesn't make sense for a probation user to see the OMU caseload. Perhaps if we need COMs to see active licences later we will have a different service call

Reply:
I think COMs do need to see ACTIVE cases and its on the Vary caseload?

Reply:
This is handled by the `getVaryCaseload` function in this case